src/user/entities/user.entity.ts

```diff
@@ -24,4 +32,10 @@ export class User {
   @Unique('email', ['email'])
   @Column({ length: 200 })
   email: string;
+
+  @CreateDateColumn({ name: 'created_at', nullable: true })
```

Comment:
```suggestion
  @CreateDateColumn({ nullable: true })
```
@Saad-Amjad It's okay to use `camelCase` for column names in our tables. 
We are doing the same for all other existing columns as well. :bow:

Reply:
Also @Saad-Amjad I just realized, we will need to create `migration` files for this change.
Let me ask @sivanpayyadakath to take over this PR. :bow: